modules/ecs6-class/line.js

```diff
@@ -2,14 +2,20 @@ const Point = require("./point");
 
 class Line {
     constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {
+        if ((typeof (n) !== 'number' && typeof (n) !== 'undefined') || (typeof (slope) !== 'number' && typeof (slope) !== 'undefined'))
+            throw Error('n or slope are not a number')
```

Comment:
I need to know exactly what the error is,  not with **or**

Reply:
I changed what needed to be changed.